### PR TITLE
New home page style

### PR DESCRIPTION
**Pull request by justindiaw**

### CODE REVIEW

File: src/app/home/home.component.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <div class = "home-wrapper">
-  <!--<div class="cover" id="top-cover">
-    <img id="cover-image" src="../../app/images/image1.jpg">
-  </div>-->
   <div class="action">
-    <h2 id="home-page-title" class="home-title" style="padding: 30px " >Where<br>Tech Volunteers Meet Nonprofit</h2>
+    <!--<h2 id="home-page-title" class="home-title" style="padding: 30px " >Where<br>Tech Volunteers Meet Nonprofit</h2>-->
+    <div id = "aniSlogan">
+      It doesn't matter if you<br> code for <span id = "aniWord" [@toggleColor]="wordColorIndex">{{aniWord}}</span><span [@cursorFlash]="cursorState">&nbsp;_</span><br>Go for a try!
```
Comment (russon77) [inline src/app/home/home.component.html]: suggestion: "go for a try" -> "get started now"

File: src/app/home/home.component.html [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <div class = "home-wrapper">
-  <!--<div class="cover" id="top-cover">
-    <img id="cover-image" src="../../app/images/image1.jpg">
-  </div>-->
   <div class="action">
-    <h2 id="home-page-title" class="home-title" style="padding: 30px " >Where<br>Tech Volunteers Meet Nonprofit</h2>
+    <!--<h2 id="home-page-title" class="home-title" style="padding: 30px " >Where<br>Tech Volunteers Meet Nonprofit</h2>-->
```
Comment (russon77) [inline src/app/home/home.component.html]: please remove commented code, git will remember

**Comment by russon77** [comment]
the images (and new logos) look *amazing*, but they are too large - 5.0MiB, 3.1MiB - ideally the home page images should not be over a few hundred KB.

**Comment by russon77** [comment]
i also think the navscroll directive needs to be extended downward (i.e. increased in height). currently it does not cover the entire logo, nor the on-hover effects for the header menu items.

otherwise, great job! looks amazing :+1:

**Comment by justindiaw** [comment]
@russon77 The I design the navscroll to be like that. Does it look bad? It's easy, just redo the size for it.

**Comment by russon77** [comment]
@justindiaw yeah it looks great, i am suggesting the height is increased to cover the logo / hover effects of the header links. check it out yourself and let me know.

**Comment by justindiaw** [comment]
@russon77 I'll pull again. Won't be too long

**Comment by justindiaw** [comment]
@russon77 Thanks for your review. And the image has been sized down, and small changes according to your review is updated to this pull request. However, I still would like the logo exceeding nav area. How about let more people give some advice to it?

**Pull request merged**